lib/rules/jsx-indent.js

```diff
@@ -77,20 +75,16 @@ module.exports = {
      * Responsible for fixing the indentation issue fix
      * @param {ASTNode} node Node violating the indent rule
      * @param {Number} needed Expected indentation character count
-     * @param {Number} gotten Indentation character count in the actual node/code
      * @returns {Function} function to be executed by the fixer
      * @private
      */
-    function getFixerFunction(node, needed, gotten) {
-      if (needed > gotten) {
-        var spaces = indentChar.repeat(needed - gotten);
```

Comment:
The old code used `String.prototype.repeat()` which is ES2015. I can switch back to using that instead of the `join()` trick I used, if it's a safe to assume repeat is supported.